python/tank/util/version.py

```diff
@@ -205,20 +244,15 @@ def _compare_versions(a, b):
         # comparing against HEAD - our version is always old
         return False
 
-    if a.startswith("v"):
-        a = a[1:]
-    if b.startswith("v"):
-        b = b[1:]
+    a = normalize_version_format(a)
+    b = normalize_version_format(b)
 
     if "packaging" in sys.modules:
```

Comment:
I don't understand why we still keep those condition. I though we already auto detected everything in the topd of the module with `version_parse`